src/steps/check-marketo-munchkin.ts

```diff
@@ -0,0 +1,35 @@
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, RunStepResponse, FieldDefinition, StepDefinition } from '../proto/cog_pb';
+
+export class CheckMarketoMunchkin extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Check that Marketo Munchkin Tracking Loads';
+  protected stepExpression: string = 'the tracking code for munchkin account id (?<id>[a-zA-Z0-9\-]+) should load';
+  protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
+  protected expectedFields: Field[] = [{
+    field: 'id',
+    type: FieldDefinition.Type.STRING,
+    description: "Munchkin Account ID associated with the user's Marketo instance (e.g. 460-tdh-945)",
+  }];
+
+  async executeStep(step: Step): Promise<RunStepResponse> {
+    const stepData: any = step.getData().toJavaScript();
+    const id: string = stepData.id;
+
+    // Navigate to URL.
+    try {
+      const actual = await this.client.getFinishedRequests();
+      if (!actual.map(request => request.url).includes('https://munchkin.marketo.net/munchkin.js')) {
```

Comment:
I've noticed that some sites seem to load specific versions of the script rather than this particular URL (e.g. `https://munchkin.marketo.net/munchkin.js?v=1.0.1`  Perhaps we should make the check here a little friendlier (e.g. if any URL in a network request contains `munchkin.marketo.net/munchkin.js`, which would also potentially account for `http`).